src/pages/DiagramsPage.vue

```diff
@@ -3,6 +3,7 @@
     id="diagrams-page"
```

Comment:
keep diagramsPage in commit `Add delete diagram`. And rename to `Add delete diagram action`

Reply:
done